src/components/Toast/index.stories.js

```diff
@@ -23,11 +23,13 @@ stories.add('Playground', () => {
         position={'top'}
         onClose={onClose}
       >
-        {"I'm a subtitle"}
+        {"I'm a subtitle with a "}
+        <a href='https://example.com'>link</a>
```

Comment:
In this case it's not critical, but be sure to update the `<Code />` examples when you update a story's code

Reply:
Ah good catch, forgot about that, thanks 🤗